core/edr-core/src/test/java/org/eclipse/tractusx/edc/edr/core/EdrCoreExtensionTest.java

```diff
@@ -42,9 +43,9 @@ void setUp(ObjectFactory factory, ServiceExtensionContext context) {
     @Test
     void shouldInitializeTheExtension(ServiceExtensionContext context) {
         extension.initialize(context);
-        
-        var service = extension.adapterTransferProcessService();
-        assertThat(service).isInstanceOf(EdrServiceImpl.class);
+
+        var service = context.getService(EdrManager.class);
```

Comment:
it can be passed through method parameters (it's injected by the DependencyInjectionExtension